src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -177,11 +167,9 @@ case class JesPendingExecutionHandle(backendCall: JesBackendCall,
                                      run: Run,
                                      previousStatus: Option[RunStatus]) extends ExecutionHandle {
   override val isDone = false
-
-  override def result = FailedExecution(new IllegalStateException)
+  override val result = NonRetryableExecution(new IllegalStateException)
```

Comment:
I imagine this would be better as `override val result = throw new NotImplementedException("BAD")` since it should be a programmer error to reach here.

As it is, if we ever reach here, the execution handle reports that it failed in a non-retryable way which isn't really accurate